config/model_configs/les_isdac_box.yml

```diff
@@ -0,0 +1,32 @@
+# device: CPUSingleThreaded
+job_id: "les_isdac_box"
+initial_condition: "ISDAC" 
+subsidence: "ISDAC" 
```

Comment:
Does ISDAC have subsidence? I didn't see it in this PR. If not I think it would be better to remove this line.

Reply:
Here it is:

https://github.com/CliMA/ClimaAtmos.jl/pull/3025/files#diff-ab240b0d863b064ad1bd28f7844952c3809af57b3424ed31794a4835f6349e47R319-R320